keras_cv/datasets/pascal_voc/load.py

```diff
@@ -49,9 +53,9 @@ def load(
     split,
     bounding_box_format,
     batch_size=None,
-    shuffle=True,
+    shuffle=None,
     shuffle_buffer=None,
-    img_size=(512, 512),
+    img_size=None,
```

Comment:
IMHO, providing a `img_size` around `Resizing` is not a very useful argument, we might instead want to provide arguments around `RandomCropResize` (or whatever the name is)

Reply:
To be honest, I'm considering just dropping the img_size argument altogether

Reply:
It feels like we mght want to just augment images indiviudally and use `tf.data.padded_batch`

Reply:
Although; it is nice for visualization.

Reply:
I think we should probably allow full customization for KPLs and probably don't even need a public data loader.